scripts/ont_generate_samplesheet.py

```diff
@@ -148,6 +153,104 @@ def make_samplesheet(currentStep):
     return file_name
 
 
+def make_samplesheet_for_qc(currentStep):
+    timestamp = dt.now().strftime("%y%m%d")
+
+    # Start by differentiating file outputs from measurements outputs
+    measurements = []
+    files = []
+
+    sample_pattern = re.compile("P\d{5}_\d{3,4}")
+
+    for art in currentStep.all_outputs():
```

Comment:
Maybe a comment here explaining the expected names of measurements and files respecitvely?

Reply:
[0a42f42](https://github.com/SciLifeLab/scilifelab_epps/pull/254/commits/0a42f42b3848271b1e9a74c0724ca1d65618e1ee)